src/us/kbase/workspace/database/provenance/ProvenanceAction.java

```diff
@@ -701,12 +701,11 @@ private List<String> processReferences(
 			}
 		}
 		
-		/** Build the {@link ProvenanceAction}. At least one field must be set before the build
-		 * attempt.
-		 * @return the provenance action.
+		/** Check the builder has any fields set.
+		 * @return true if no fields are set, false otherwise.
 		 */
-		public ProvenanceAction build() {
-			if (	// ew
```

Comment:
hurrah! no more `ew`

Reply:
I actually was disappointed when I noticed I removed that. It's still ew in essence, if no longer by name